suseconnect-ng.spec

```diff
@@ -36,6 +36,7 @@ BuildRequires:  zypper
 BuildRequires:  ruby-devel
 %if %{with hwinfo}
 %global test_hwinfo_args -test-hwinfo
+
 # packages required only for hwinfo tests
 %ifarch %ix86 ia64 x86_64 %arm aarch64
```

Comment:
`%ifarch %ix86` when further down you Exclude %ix86 seems like some cleanup could be done

Reply:
Updated to skip %ix86.